swagger.yaml

```diff
@@ -2771,18 +2848,25 @@ components:
           example: 2022-09-14T16:35:31Z
     AppliedCoupon:
       type: object
+      required:
+        - applied_coupon
       properties:
         applied_coupon:
           $ref: '#/components/schemas/AppliedCouponObject'
     AppliedCoupons:
       type: object
+      required:
+        - applied_coupons
       properties:
         applied_coupons:
           type: array
           items:
             $ref: '#/components/schemas/AppliedCouponObject'
     AppliedCouponInput:
       type: object
+      required:
+        - external_customer_id
+        - coupon_code
```

Comment:
Here we should have only `applied_coupon` but under applied coupon there should be those two parameters

Reply:
exactly but this code chunk is not on correct place, it should be places under `applied_coupon`